__std__/nix/toolchain/packages/hie-bios.nix

```diff
@@ -1,5 +1,5 @@
 { inputs, cell }:
 
-# TODO(std) we need haskell-nix for this
+# TODO(std) is this needed?
```

Comment:
it's occasionally useful for debugging HLS issues, but perhaps not

Reply:
doesn't hurt to keep 👍